Datadog.Trace.sln

```diff
@@ -1607,26 +1608,143 @@ Global
 		SolutionGuid = {160A1D00-1F5B-40F8-A155-621B4459D78F}
 	EndGlobalSection
 	GlobalSection(SharedMSBuildProjectFiles) = preSolution
+		tracer\test\test-applications\Samples.Shared\Samples.Shared.projitems*{021efba6-c4ba-4de5-bf3f-c263ee9e20db}*SharedItemsImports = 5
+		tracer\test\test-applications\Samples.Shared\Samples.Shared.projitems*{061ab58b-8235-4dae-8d56-5f081dd78f5e}*SharedItemsImports = 5
```

Comment:
I don't think these should all be added here 🤔

Reply:
Visual Studio rewrites 🤔, I've added a reference to the new sample (which in turn references the `Samples.Shared` proj)